_includes/recent_waiter.js

```diff
@@ -0,0 +1,70 @@
+;(function () {
+  const RECENT_QUERY_PARAM = 'recent'
+  const RECENT_PER_PAGE = 9
+
+  const template = `<div class="card">
```

Comment:
This template here is a nice template to export actually but it only holds a placeholder card.  Looks nice, has no content, and maybe could get a fixed height to match the usual height of a rendered full card better.

Reply:
Any reason you did not re-use the template element that already exists? Or perhaps we replace that with this?

Reply:
This is a real placeholder card.  It looks nice but doesn't have all the branches/elements we actually have.  The card-template has all the possible slots/div/spans we can eventually fill out with data.  I couldn't use card.js with it as that is not yet loaded.  There might be ways to reduce the card template by removing element branches from it, but really you could design any placeholder, just something that looks nice.  The fancy sites usually animate them to mimic a waiting state...